CHANGELOG.md

```diff
@@ -0,0 +1,188 @@
+# Changelog
+
+## v1.7.5
+
+- 🔊[RUMF-408] add new session check logs ([#318](https://github.com/DataDog/browser-sdk/pull/318))
+- [RUMF-407] improve resource timings collection ([#315](https://github.com/DataDog/browser-sdk/pull/315))
+- 🔧 improve CBT test names ([#314](https://github.com/DataDog/browser-sdk/pull/314))
+- [RUMF-382] prepare support for multiple feature flags ([#312](https://github.com/DataDog/browser-sdk/pull/312))
+- 🔧 update cbt chrome mobile ([#313](https://github.com/DataDog/browser-sdk/pull/313))
+- v1.7.5 ([#311](https://github.com/DataDog/browser-sdk/pull/311))
```

Comment:
This commit is probably not needed

Reply:
Yep but those will have to be manually removed.